packages/ember-metal/lib/weak_map.js

```diff
@@ -8,8 +7,14 @@ import {
 let id = 0;
 function UNDEFINED() {}
 
+// Returns whether Type(value) is Object according to the terminology in the spec
+function isObject(value) {
+  return (typeof value === 'object' && value !== null) || typeof value === 'function';
+}
+
 /*
- * @private
+ * @public
```

Comment:
I think the YUIDoc commands have to be after the description, can you move these down a bit?